apps/etterlatte-saksbehandling-ui/client/src/components/etteroppgjoer/revurdering/EtteroppgjoerRevurderingResultat.tsx

```diff
@@ -0,0 +1,41 @@
+import { useEtteroppgjoer } from '~store/reducers/EtteroppgjoerReducer'
+import { Box, HStack, Label, VStack } from '@navikt/ds-react'
+import { EtteroppgjoerResultatType } from '~shared/types/EtteroppgjoerForbehandling'
+import { CurrencyExchangeIcon } from '@navikt/aksel-icons'
+
+const ResultatVisning = ({ tekst }: { tekst: string }) => (
+  <HStack gap="2" maxWidth="fit-content">
+    <CurrencyExchangeIcon fontSize="1.5rem" aria-hidden />
+    <VStack gap="2" maxWidth="42.5rem" marginBlock="05 0">
+      <Label>{tekst}</Label>
+    </VStack>
+  </HStack>
+)
```

Comment:
Denne er vel det samme for både forbehandling/revurdering. Kan kanskje ligge i en egen fil

Reply:
Jepp!